### ISSUE TITLE
Decide on maintenance of Campanulaceae and Tetrapoda examples

### ISSUE THREAD

**Issue by hlapp** [phyloref/phyloref-ontology#53]
Both are currently badly outdated, and don't comply with the recent phyloref ontology model. We need to decide whether to update or delete them, given that we are currently consolidating on the gators ontology as the main and well-maintained example.

@gaurav and @ncellinese thoughts?

**Comment by gaurav** [comment]
I don't think there's a lot of value in updating them. There might be some value in keeping them around as a record of how phyloreferences evolved over time, as long as they're moved into a separate folder and clearly described as deprecated. Otherwise, I think it makes sense to delete them.

**Comment by hlapp** [comment]
> There might be some value in keeping them around as a record of how phyloreferences evolved over time

Note that this in essence describing the purpose of version control, which we _are_ using. So for keeping a historical record, I'm inclined to say version control addresses that sufficiently well. It would really need to be a commitment to maintain them going forward that would justify keeping them fully visible as part of the _present_ record.

**Comment by gaurav** [comment]
Fair enough. In that case, I think it makes more sense to delete them rather than to update them -- there's an interesting apomorphy-based example in the Tetrapoda example that might be useful in a future project, but I don't think there's anything in the Campanulaceae example that isn't already in the Gators example.
